src/Featurizers/PCAFeaturizer.h

```diff
@@ -43,16 +48,13 @@ class PCATransformer : public StandardTransformer<MatrixT, MatrixT> {
 
     void save(Archive &ar) const override;
 
-    std::ptrdiff_t getEigenVectorRowsNumber() const;
```

Comment:
Dmitri wants shape information to be exposed, so that I added these two funcs

Reply:
Looks like a merge problem - I will figure out what happened and update the PR.